{{cookiecutter.project_repo_name}}/.travis.yml

```diff
@@ -1,14 +1,21 @@
 dist: xenial   # required for Python >= 3.7
 language: python
+os:
+  - linux
 python:
   - "3.6"
   - "3.7"
+  - "3.8"
```

Comment:
Currently PyWPS does not work with Python 3.8 ... should we allow 3.8 to fail in travis?